addons/binding/org.openhab.binding.irtrans/src/main/java/org/openhab/binding/irtrans/handler/EthernetBridgeHandler.java

```diff
@@ -779,13 +778,13 @@ protected void onWritable(ByteBuffer buffer) {
 
     protected void onRead(ByteBuffer byteBuffer) {
         try {
-            logger.trace("Received bytebuffer : '{}'", DatatypeConverter.printHexBinary(byteBuffer.array()));
+            logger.trace("Received bytebuffer : '{}'", HexUtils.bytesToHex(byteBuffer.array()));
```

Comment:
The `HexUtils.bytesToHex()` method is applied every time. Should we encapsulate it by an `if (logger.isTraceEnabled())`?